scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -1009,22 +1008,28 @@ public Void executeWork(Session session) {
                 long jobId = jobId(job);
 
                 String jobUpdate = "update JobData set status = :status, " +
-                        "numberOfPendingTasks = :numberOfPendingTasks, " +
-                        "numberOfRunningTasks = :numberOfRunningTasks where id = :jobId";
+                    "numberOfPendingTasks = :numberOfPendingTasks, " +
+                    "numberOfRunningTasks = :numberOfRunningTasks, " +
+                    "numberOfFailedTasks = :numberOfFailedTasks, " +
+                    "numberOfFaultyTasks = :numberOfFaultyTasks, " +
+                    "numberOfInErrorTasks = :numberOfInErrorTasks " + "where id = :jobId";
 
                 JobInfo jobInfo = job.getJobInfo();
 
                 session.createQuery(jobUpdate).setParameter("status", jobInfo.getStatus())
                         .setParameter("numberOfPendingTasks", jobInfo.getNumberOfPendingTasks())
                         .setParameter("numberOfRunningTasks", jobInfo.getNumberOfRunningTasks())
+                        .setParameter("numberOfFailedTasks", jobInfo.getNumberOfFailedTasks())
+                        .setParameter("numberOfFaultyTasks", jobInfo.getNumberOfFaultyTasks())
+                        .setParameter("numberOfInErrorTasks", jobInfo.getNumberOfInErrorTasks())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "numberOfInErrorTasks" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)